packages/admin/admin/src/mui/menu/Menu.styles.ts

```diff
@@ -57,7 +60,15 @@ export type MenuClassKey = "drawer" | "permanent" | "temporary" | "open" | "clos
 
 export const styles = () => {
     return createStyles<MenuClassKey, MenuProps>({
-        drawer: {},
+        drawer: {
+            "& > div": {
+                "&::-webkit-scrollbar": {
+                    display: "none",
+                },
+                scrollbarWidth: "none",
+                msOverflowStyle: "none",
```

Comment:
Because it changes the internal width of the objects and for something like a navbar i find it very weird and annoying if that happens.


https://github.com/vivid-planet/comet/assets/56400587/c7591d32-7f12-4905-9720-d36148728670

Reply:
✅